frontend/src/routes/Governance/overview/Overview.tsx

```diff
@@ -349,7 +349,7 @@ function ClustersCard() {
         <ExpandableSection
           toggleContent={
             <div>
-              <span style={{ whiteSpace: 'pre' }}>{`${t(`Show {{count}} more`, { count: xtraList.length })}  `}</span>
+              <span style={{ whiteSpace: 'pre' }}>{`${t('Show {{count}} more', { count: xtraList.length })}  `}</span>
```

Comment:
```suggestion
              <span style={{ whiteSpace: 'pre' }}>{t('Show {{count}} more', { count: xtraList.length })}</span>
```
I would remove the outer template string. It's not needed.

Reply:
Thanks Kevin!